R/Analysis.R

```diff
@@ -224,5 +226,13 @@ MakeEnrichmentDotPlot <- function(seuratObj,
     ylab(yField) +
     guides(fill = guide_colorbar(order = 1))
   
+  if(!is.null(facetingField)){
+    if(!(facetingField %in% extraGroupingFields)){
```

Comment:
why wouldnt we do this test upfront, and add it to extraGroupingFields for the user?